ucontrib/Platform/fst/Platform.Bytes.fst

```diff
@@ -78,7 +78,7 @@ let cbyte b = Seq.index b 0
 val cbyte2 : b:bytes{Seq.length b >= 2} -> Tot (byte * byte)
 let cbyte2 b = (Seq.index b 0, Seq.index b 1)
 
-val index : b:bytes -> i:nat{Seq.length b >= i} -> Tot byte
+val index : b:bytes -> i:nat{Seq.length b > i} -> Tot byte
 let index b i = Seq.index b i
```

Comment:
How did this code ever go through? In particular, the `Seq.index` shouldn't have typechecked with the incorrect refinement.